### PR TITLE
Default is no credentials required

### PR DESCRIPTION
**Pull request by aledsage**

### CODE REVIEW

File: guide/ops/server-cli-reference.md [hunk 1]
```diff
@@ -15,8 +15,9 @@ To launch Brooklyn, from the directory where Brooklyn is unpacked, run:
 {% endhighlight %}
 
 With no configuration, this will launch the Brooklyn web console and REST API on [`http://localhost:8081/`](http://localhost:8081/).
-No password is set, but the server is listening only on the loopback network interface for security.
-Once [security is configured](brooklyn_properties.html), Brooklyn will listen on all network interfaces by default.
+listening on all network interfaces. No credentials are required by default. It is strongly
```
Comment (neykov) [inline guide/ops/server-cli-reference.md]: Remove preceding dot, don't break line.

**Comment by neykov** [comment]
LGTM, apart from changing a dot to a comma.

File: guide/start/running.md [hunk 2]
```diff
@@ -134,13 +134,15 @@ Extract this zip file to a directory on your computer such as `c:\Program Files\
 
 ---
 
-It is not necessary at this time, but depending on what you are going to do, 
-you may wish to set up other configuration options first:
+By default, no authentication is required and the web-console will listen on all network interfaces.
+For a production system, or if Apache Brooklyn is publicly reachable, it is strongly recommended
```
Comment (mikezaccardo) [inline guide/start/running.md]: Perhaps add these justifications to the files above which also recommend the use of security.

**Comment by aledsage** [comment]
Thanks - changes incorporated. Merging now, and will cherry-pick into 0.10.0 branch (to push these changes to the live docs site).

**Pull request merged**